integrations/applications/nextjs-application/resources/info.json

```diff
@@ -0,0 +1,23 @@
+{
+    "id": "nextjs-application",
+    "version": "1.0.0",
+    "name": "Next.js",
+    "description": "Server rendered or static websites scaffolded with Create Next App or any other tool.",
+    "image": "${CONSOLE_BASE_URL}/resources/applications/assets/images/technologies/nextjs-logo.svg",
+    "displayOrder": 0,
+    "category": "TECHNOLOGY",
+    "tags": [
+        "OIDC"
+    ],
+    "type": "applications",
+    "customAttributes": [
+        {
+            "key": "featureStatus",
+            "value": "new"
+        },
+        {
+            "key": "useNewCreateWizard",
```

Comment:
We don't need to use the `useNewCreateWizard` configuration, it's only required for templates in the `DEFAULT` category.